torch/nn/parallel/distributed.py

```diff
@@ -567,6 +583,8 @@ def __setstate__(self, state):
             param_to_name_mapping = {}
         # Builds reducer
         self._ddp_init_helper(parameters, expect_sparse_gradient, param_to_name_mapping)
+        if self.static_graph:
+            self._set_static_graph()
```

Comment:
if `find_unused_parameters=True` is set in ddp, should we add a warnings.warn once indicating that the flag will have no effect with static graph training?

Reply:
Just realized that this path runs when ddp model is saved/loaded. We can add a warning here as well as in the regular init path